apps/cloudappclient/app.js

```diff
@@ -21,7 +22,7 @@ module.exports = activity => {
   // playerId manager version 1
   var pm = new PlayerManager()
   // skill os
-  var sos = new Manager(directive, Skill)
+  var sos = new Manager(directive, Skill, eventRequest)
```

Comment:
EventRequest should be stateless, why do we need to pass the object to Manager to use?

Reply:
Because I need to use a simulated eventrequest, so I can write unit tests.

Reply:
Writing unit tests are good, but I'm -1 on breaking another good design to let something testable. Why not just making this parameter are optional then we could only pass it at test code.